collect_app/src/main/java/org/odk/collect/android/tasks/sms/SmsUtils.java

```diff
@@ -0,0 +1,160 @@
+package org.odk.collect.android.tasks.sms;
+
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+
+import org.odk.collect.android.R;
+import org.odk.collect.android.events.SmsEvent;
+import org.odk.collect.android.tasks.sms.models.MessageStatus;
+import org.odk.collect.android.tasks.sms.models.SmsProgress;
+import org.odk.collect.android.tasks.sms.models.SmsSubmission;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
+public class SmsUtils {
+
+    public static final String SMS_SEND_ACTION = "org.odk.collect.android.COLLECT_SMS_SEND_ACTION";
+    public static final String SMS_DELIVERY_ACTION = "org.odk.collect.android.COLLECT_SMS_DELIVERY_ACTION";
+    public static final String SMS_INSTANCE_ID = "COLLECT_SMS_INSTANCE_ID";
+    public static final String SMS_MESSAGE_ID = "COLLECT_SMS_MESSAGE_ID";
```

Comment:
Can be package-private

Reply:
Didn't make them package private because they won't be accessible from the tests.